code/CapstoneBackend/Model/User.cs

```diff
@@ -29,5 +34,13 @@ public class User
         ///     The last name of the user
         /// </summary>
         public string LastName { get; set; } = string.Empty;
+
+        public IList<Trip> Trips => this.getTrips();
+
+        private IList<Trip> getTrips()
```

Comment:
Should this be a property in the model class?
I was of the understanding that we should be retrieving data using the helper functions of the Manager classes, instead of data models